test/azure/Expected/AcceptanceTests/Lro/lro/aio/operations_async/_lr_os_custom_header_operations_async.py

```diff
@@ -106,6 +106,7 @@ async def put_async_retry_succeeded(
         :param product: Product to put.
         :type product: ~lro.models.Product
         :keyword callable cls: A custom type or function that will be passed the direct response
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state
```

Comment:
I think our codegen templates should add trailing "." to the end of kwarg descriptions.

Reply:
fixed